File: Java/OOP/project1/src/project/Employee.java
package project;

abstract class Employee extends Person {
    private int salary;




        
        
        public int getSalary() {
        	return salary;
             }
        
        public void setSalaty(int salary) {
        	this.salary = salary;
        	
        }
    
    public Employee() {

    }
    public Employee(int age, int salary) { super (age) ;
        //this.age = age; toto nahrazuje dìdìní konstruktoru base(age)
        this.salary = salary;
    }



    public void writeInfo() {
        //base.writeInfo();
        System.out.println(age);
        System.out.print(" plat zamìstnance je "+  getSalary() + " Kè " + GetAge()+" "+  GetCount() );  // get/set
    }
}

File: C#/OOP/Prvni/Employee.cs
using System;

//v tomhle jsem měl trochu bordel, kdyžtak checknout doma

namespace Prvni12 {
    abstract class Employee : Person {
        private int salary;



        public int Salary {
            get { return salary; }
            set {
                salary = value;
            }
        }
        public Employee() {

        }
        public Employee(int age, int salary) : base(age) {
            //this.age = age; toto nahrazuje dědění konstruktoru base(age)
            this.Salary = salary;
        }



        public override void writeInfo() {
            //base.writeInfo();
            Console.Write(age);
            Console.Write($" plat zaměstnance je  {Salary} Kč {GetAge()} {GetCount()} ");  // get/set
        }
    }
}


